runelite-client/src/main/java/net/runelite/client/plugins/blastmine/BlastMinePlugin.java

```diff
@@ -81,26 +74,18 @@ BlastMinePluginConfig getConfig(ConfigManager configManager)
 	@Subscribe
 	public void onGameObjectSpawned(GameObjectSpawned event)
 	{
-		GameObject gameObject = event.getGameObject();
+		final GameObject gameObject = event.getGameObject();
+		final BlastMineRock newRock = new BlastMineRock(gameObject);
+		final BlastMineRock oldRock = rocks.get(gameObject.getWorldLocation());
 
-		if (BlastMineRockType.getRockType(gameObject.getId()) == null)
```

Comment:
This was done based on Adam's review on the other PR: https://github.com/runelite/runelite/pull/2308#discussion_r186268888